src/templates/legislation-changes/index.hbs

```diff
@@ -13,7 +13,7 @@
       <div>
         <h1>The latest in pension transfer news</h1>
         <div>
-          <a href="{{formLink}}" title="{{formTitle}}" class="button big">Keep me informed</a>
+          <a href="/keep-me-informed" title="formTitle" class="button big">Keep me informed</a>
```

Comment:
In the Gruntfile I declare a variable called "formLink". Once declared you can use them in your templates with Handlebar placeholder like so: `{{formLink}}`. It's one way to do DRY coding. Because we link to this page from various places it could be hard to keep track. If ever we change those links, we might forget to update one without knowing. That is another reason to only store that URL and title in only one place.

Leave this for now. Just in future we'll organise it as such.